docs/modules/ROOT/pages/ai-services.adoc

```diff
@@ -168,7 +168,7 @@ quarkus.langchain4j.huggingface.m2.api-key=sk-...
 
 As LLMs are stateless, the memory — comprising the interaction context — must be exchanged each time. To prevent storing excessive messages, it's crucial to evict older messages.
 
-The `chatMemoryProviderSupplier` attribute of the `@RegisterAiService` annotation enables configuring the `dev.langchain4j.memory.chat.ChatMemoryProvider`. The default value of this annotation is `RegisterAiService.BeanIfExistsChatMemoryProviderSupplier.class`
+The `chatMemoryProviderSupplier` attribute of the `@RegisterAiService` annotation enables configuring the `dev.langchain4j.memory.chat.ChatMemoryProvider`. The default value of this annotation is `RegisterAiService.BeanChatMemoryProviderSupplier.class`
 which means that the `AiService` will use whatever `ChatMemoryProvider` bean is configured by the application.
```

Comment:
, or use the default one provided by the extension?

Reply:
Ah never mind, now I see what you mean.